modules/virt-accessing-vmi-yaml-ssh.adoc

```diff
@@ -0,0 +1,133 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-accessing-vm-consoles.adoc
+
+[id="virt-accessing-vmi-yaml-ssh_{context}"]
+= Accessing a virtual machine instance via SSH with YAML configurations
+
+You can enable a SSH connection to a virtual machine instance (VMI) without the need to run the `virtctl expose` command. When the YAML file for the VMI and the YAML file for the service are configured and applied, the service forwards the SSH traffic to the VMI.
+
+The following examples show the configurations for the VMI's YAML file and the service YAML file.
+
+.Prerequisites
+* Install the OpenShift CLI (`oc`).
+* You must be in the same project as the VMI.
+
+.Procedure
+. In the YAML file for the VMI, add the label and a value for exposing the service for SSH connections. Enable the `masquerade` feature for the interface.
+
++
+.Example output
+[source,yaml]
+----
+...
+kind: VirtualMachineInstance
+apiVersion: kubevirt.io/v1
+kind: VirtualMachineInstance
```

Comment:
Again - the manifest should be of a VirtuaMachine, not VirtuaMachineInstance.
@ousleyp , please correct me if I am wrong here (and if I do - I apologize for the noise).

Reply:
@yossisegev I do remember when we decided to only document VMs and not VMIs, but IIRC, after that point we had stories to add VMI-specific information to the docs. (I think that @lmandavi had these stories, so she might remember better than I do.)

My understanding of the consensus at this point is that it is best to refer to VMs only, unless we *need* to talk about VMIs for a given procedure.

Reply:
When virtual machine instances feature was introduced last year, I worked extensively with team members on that effort, and the term VMI was used to refer to virtual machine instances in our documentation set. 

I don't recall learning that we we avoid using virtual machine instances in yaml examples. @phoracek @myakove @asadan@redhat.com - please advise if the yaml examples need to be corrected to show virtual machines and not virtual machine instances? If that is the case, our doc team needs to make a note of this for content developed in the future. Thanks.

Reply:
Changed all VMI references to VM. This preference should be noted by the cnvdoc team for future content development - I was unaware of the preference.